seed/challenges/01-front-end-development-certification/intermediate-bonfires.json

```diff
@@ -39,6 +39,12 @@
         "El número menor no siempre será el primer elemento en el arreglo.",
         "Recuerda utilizar <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Read-Search-Ask</a> si te sientes atascado. Intenta programar en pareja. Escribe tu propio código."
       ],
+      "namePtBR": "Some Todos os Números em um Intervalo",
+      "descriptionPtBR": [
```

Comment:
It should be just `Pt`. At the moment we are not supporting multiple dialects. In the future if we decide to, it will be a simple to migrate.